devstack/lib/nfp

```diff
@@ -213,6 +213,8 @@ function create_nfp_gbp_resources {
 
     neutron router-gateway-clear\
```

Comment:
@AkashDeepSrivastava - Don't we need to remove this router-gateway-clear command?